providers/zendesksupport/read.go

```diff
@@ -89,3 +95,190 @@ func formatStartTime(config common.ReadParams) string {
 
 	return strconv.FormatInt(unixTime, 10)
 }
+
+// requestCustomFields makes and API call to get model describing custom fields.
+// For not applicable objects the empty mapping is returned.
+// The mapping is between "custom field id" and struct containing "human-readable field name".
+//
+// Custom fields are always associated with "ticket_fields" regardless of the object type.
```

Comment:
might make it easier to read if we separate this out into a `customFields.go` file